src/Forms/Input.js

```diff
@@ -22,7 +22,30 @@ const Input = ({label, name, type}) => (
   </FormContext.Consumer>
 );
 
+export const Select = ({ name, label, restrictEndDate }) => (
+  <FormContext.Consumer>
+    {({data, handleChange, handleEndDate}) => (
+      <React.Fragment>
+        <p style={{fontSize: '.75rem', color: 'rgba(0, 0, 0, 0.54)'}}>{label}</p>
+        <select
+          value={data[name].value}
+          onChange={(e) => {
+            handleChange(e);
+
+            if (restrictEndDate) {
+              handleEndDate(e.target)
+            }
+          }}
+          name={name}
+          style={{fontSize: '1rem', width: '100%'}}
+        >
+          {data[name].options
+            .map((option, i) => <option value={option.value} key={i}>{option.display}</option>)}
```

Comment:
It's considered an antipattern to use index as a key. I thank you can just do `key={option.value}`